src/hevm/hevm-cli/hevm-cli.hs

```diff
@@ -389,31 +389,30 @@ findJsonFile Nothing = do
         ]
 
 equivalence :: Command Options.Unwrapped -> IO ()
-equivalence cmd = undefined
-  {-
-  do let bytecodeA = hexByteString "--code" . strip0x $ codeA cmd
-         bytecodeB = hexByteString "--code" . strip0x $ codeB cmd
-     maybeSignature <- case sig cmd of
-       Nothing -> return Nothing
-       Just sig' -> do method' <- functionAbi sig'
-                       return $ Just (view methodSignature method', snd <$> view methodInputs method')
-
-     void . runSMTWithTimeOut (solver cmd) (smttimeout cmd) (smtdebug cmd) . query $
-       equivalenceCheck bytecodeA bytecodeB (maxIterations cmd) (askSmtIterations cmd) maybeSignature >>= \case
-         Cex vm -> do
-           io $ putStrLn "Not equal!"
-           io $ putStrLn "Counterexample:"
-           showCounterexample vm maybeSignature
-           io exitFailure
-         Qed (postAs, postBs) -> io $ do
-           putStrLn $ "Explored: " <> show (length postAs)
-                       <> " execution paths of A and: "
-                       <> show (length postBs) <> " paths of B."
-           putStrLn "No discrepancies found."
-         Timeout () -> io $ do
-           hPutStr stderr "Solver timeout!"
-           exitFailure
-      -}
+equivalence cmd = do
+  let bytecodeA = hexByteString "--code" . strip0x $ codeA cmd
+      bytecodeB = hexByteString "--code" . strip0x $ codeB cmd
+      veriOpts = VeriOpts { simp = True
+                            , debug = False
+                            , maxIter = maxIterations cmd
+                            , askSmtIters = askSmtIterations cmd
+                          }
+  -- maybeSignature <- case sig cmd of
+  --   Nothing -> return Nothing
+  --   Just sig' -> do method' <- functionAbi sig'
+                    -- return $ Just (view methodSignature method', snd <$> view methodInputs method')
```

Comment:
do we need this?

Reply:
I actually checked this, and I think it was some kind of reminder that we didn't implement something and I though it may be important to keep around. I likely misremembered. I am removing.